docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Apache Kafka Broker"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure the 
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
+
+This guide assumes a 
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component), only
+`eventing-core` is required.
```

Comment:
```suggestion
[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
```

Reply:
I'm not sure if it's necessary to say just `eventing-core` is required, we assume they'll install Knative Eventing as is?

Reply:
I wanted to point out that a user doesn't need to install everything listed there but only the core, ie a user can stop the installation to the second bullet of this section: https://knative.dev/development/install/any-kubernetes-cluster/#installing-the-eventing-component